### PR TITLE
Updating swarm info

### PR DESCRIPTION
**Pull request by marians**
- Adding API endpoint output
- Renaming “Cluster status” to “API status”, as we are pinging the API, not the cluster
- Reordering to make more sense
- Usage test improvement
- Updated int-tests

## Preview with up to date CLI

```
$ SWARM_ENDPOINT_URL=https://api.giantswarm.io swarm info
Swarm CLI version:   0.22.1+git
API endpoint:        https://api.giantswarm.io
API status:          reachable
Logged in as user:   marian
Current environment: marian/dev
```

## Preview with outdated CLI

```
$ SWARM_ENDPOINT_URL=https://api.giantswarm.io swarm info
Swarm CLI version:   0.20.1+git (outdated)
API endpoint:        https://api.giantswarm.io
API status:          reachable
Logged in as user:   marian
Current environment: marian/dev

Please update your swarm CLI to the latest version 0.22.1.
Find the download link and instructions here:

    https://docs.giantswarm.io/install

```

### CODE REVIEW

**Comment by marians** [comment]
RFR @zyndiecate or @ewoutp or @ZeissS

**Comment by hectorj2f** [comment]
@marians From my point of view, the API endpoint should be unified at some point to `api.giantswarm.io`

**Comment by marians** [comment]
@hectorj2f Yes, I agree. Would leave this out of this PR though.

File: info.go [hunk 1]
```diff
@@ -12,8 +12,8 @@ import (
 var (
 	infoCmd = &cobra.Command{
 		Use:   "info",
-		Short: "Print the current settings of the swarm CLI client",
-		Long:  "Print the current settings of the swarm CLI client",
+		Short: "Print current settings and context information",
+		Long:  "Print current settings and context information of the CLI",
```
Comment (ewoutp) [inline info.go]: I'm wondering if this "longer" version really adds value compared to the short version.

File: info.go [hunk 2]
```diff
@@ -37,16 +37,28 @@ func infoCmdRun(cmd *cobra.Command, args []string) {
...
+	// our output message
+	m := ""
+
+	// Do version check
+	v, err := versioncheck.GetCliVersionInfo(projectVersion)
+	if err != nil {
+		Quitf(defaultErrorMsgTemplate, err)
+	}
+
+	m += fmt.Sprintf("Swarm CLI version:   %s%s\n", v.CurrentVersion, v.Postfix)
```
Comment (ewoutp) [inline info.go]: If we touch this anyway, we should also use `github.com/ryanuber/columnize` to to the column alignment.

File: info.go (hunk 1 above)
Comment (marians) [inline info.go]: Not really. It probably doesn't hurt either.

File: info.go (hunk 2 above)
Comment (marians) [inline info.go]: Actually I don't want to make this a refactoring. Feel like I'm the wrong person for that.

**Comment by marians** [comment]
@ewoutp To merge or not to merge?

File: info.go (hunk 2 above)
Comment (ewoutp) [inline info.go]: Fine, just make an issue then

**Comment by ewoutp** [comment]
LGTM

**Pull request merged**